app/backend/app.py

```diff
@@ -86,23 +89,26 @@ async def assets(path):
     return await send_from_directory(Path(__file__).resolve().parent / "static" / "assets", path)
 
 
-@bp.route("/content/<path>")
+@bp.route("/content/usecase/<usecase>/<path>")
```

Comment:
Waarom is het usecase/usecase en niet maar een keer usecase?

Reply:
Wanted to keep it in sync with the frontend code (and the setup at DnB). Think its also a clear identified to first have usecase/<followed_by_the_actual_use_case>       

 Frontend code: 
 
```
  {
      path: "usecase/:usecase_id",
      element: <Chat {...config} />
 }
```